packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -865,8 +866,7 @@ describe('plugin-meetings', () => {
 
         it('should not call leave() if addMedia fails the first time and succeeds the second time and should only call join() once', async () => {
           const addMediaError = new Error('fake addMedia error');
-          const leaveError = new Error('leave error');
-          const leaveStub = sinon.stub(meeting, 'leave').rejects(leaveError);
+          const leaveStub = sinon.stub(meeting, 'leave');
```

Comment:
Why have we changed this?

Reply:
I just noticed that it was not needed. Looks like when someone created this test they copied from another one where Meeting.leave() needed to reject, but in this test the SDK is not expected to call Meeting.leave() at all - see the check in line 886, so making Meeting.leave() fail is unnecessary logic that could confuse someone reading this test.